scripts/src/java/org/oppia/android/scripts/todo/TodoOpenCheck.kt

```diff
@@ -25,12 +25,19 @@ import java.io.FileInputStream
  * - path_to_json_file: path to the json file containing the list of all open issues on Github
  *
  * Example:
- *   bazel run //scripts:todo_check -- $(pwd) scripts/assets/todo_exemptions.pb open_issues.json
+ *   bazel run //scripts:todo_open_check -- $(pwd) scripts/assets/todo_open_exemptions.pb
+ *   open_issues.json
  *
  * NOTE TO DEVELOPERS: The script is executed in the CI enviornment. The CI workflow creates a
  * json file from the GitHub api which contains a list of all open issues of the
- * oppia/oppia-android repository. To execute it without the CI, please create an open issues json
+ * oppia/oppia-android repository. To execute it without the CI, please create open issues json
  * file and provide its path to the script in the format as stated above.
+ *
+ * Instructions to create the open_issues.json file:
+ * 1. Setup Github CLI Tools locally.
```

Comment:
```suggestion
 * 1. Set up Github CLI Tools locally.
```

(Set up is the verb form of the noun setup).

Reply:
Done.